### PR TITLE
set default template_variable_presets to nil

### PR DESCRIPTION
**Pull request by uthark**

### CODE REVIEW

File: lib/kennel/models/dashboard.rb [hunk 1]
```diff
@@ -103,10 +103,10 @@ def as_json
           title: "#{title}#{LOCK}",
           description: description,
           template_variables: render_template_variables,
-          template_variable_presets: template_variable_presets,
           widgets: all_widgets
         }
 
+        @json[:template_variable_presets] = template_variable_presets if template_variable_presets
```
Comment (grosser) [inline lib/kennel/models/dashboard.rb:109]: we need to set the nil because otherwise if you remove presets then it would not deleted

**Comment by grosser** [comment]
oh boy ... `+  +template_variable_presets nil -> nil`
... so I think I was wrong and it's not that it's `nil` but that it is not returned at all :(

**Comment by grosser** [comment]
needs ignore_defaults I think ... can steal from monitor.rb

**Comment by uthark** [comment]
@grosser PTAL. integration tests are now passing too.

**Comment by grosser** [comment]
1.72.2

**Pull request merged**